src/web/services/feedback-sessions.service.ts

```diff
@@ -141,7 +141,6 @@ export class FeedbackSessionsService {
         isinrecyclebin: 'false',
       };
     }
-
```

Comment:
If this change isn't required by lint, let's keep it as it is since you don't have to change this part of the function.

Reply:
Thanks for catching it! I've changed it back.